build/repo.targets

```diff
@@ -0,0 +1,18 @@
+<Project>
+    <PropertyGroup>
+        <RestoreDependsOn>$(RestoreDependsOn);MinifyFiles</RestoreDependsOn>
+        <ScriptsDir>$(RepositoryRoot)scripts</ScriptsDir>
+    </PropertyGroup>
+    <Target Name="MinifyFiles">
+        <Exec Command="node --version" ContinueOnError="true">
+            <Output TaskParameter="ExitCode" PropertyName="ErrorCode" />
+        </Exec>
+        <Error Condition="'$(ErrorCode)' != '0'" Text="Node.js is required to build and run this project. To continue, please install Node.js from https://nodejs.org/, and then restart your command prompt or IDE." />
+        <ItemGroup>
+            <SiteJsFiles Include="$(RepositoryRoot)src\Microsoft.DotNet.Web.ProjectTemplates\content\*\wwwroot\js\site.js" />
+            <SiteCssFiles Include="$(RepositoryRoot)src\Microsoft.DotNet.Web.ProjectTemplates\content\*\wwwroot\css\site.css" />
+        </ItemGroup>
+        <Exec Command="npx uglifycss %(SiteCssFiles.Identity) --output %(RelativeDir)site.min.css" />
+        <Exec Command="npx uglify-js %(SiteJsFiles.Identity) --output %(RelativeDir)site.min.js" />
```

Comment:
I'd imagine this is not super fast, as it's creating a new Node process for every `.css`/`.js` file in all the templates. It also relies on `npx` being installed globally, which is not standard within a Node environment.

Maybe it's OK, but if either aspect of this becomes problematic, we could add a small Node script that does it (so, one Node process that iterates through all the files and minifies them) and have it invoked via `node ourscript.js`.